src/providers/azurestack/crypto/mod.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2017 CoreOS, Inc.
```

Comment:
`2021`

This came up in discussion in another project as well. For this project at least, we should probably be able to drop all these blocks and just rely on the `Cargo.toml` license field. (To be clear, I mean this as a follow-up, not something you need to do here.)